lib/views/pages/home/content/map/components/map_pin_pop_up.dart

```diff
@@ -0,0 +1,74 @@
+import "package:flutter/material.dart";
+
+class MapPinPopUp extends StatelessWidget {
```

Comment:
There's a huge amount of code repetition between this and the `profile_info_pop_up.dart`. I think it would be better to make some abstract class that factorises most of the code with some parameter for the `title`, `content?` and `actions`, and make `MapPinPopUp` and `ProfileInfoPopUp` inherit from it.

Moreover, this class is missing class documentation ;)

Reply:
Fixed with [3845991](https://github.com/ProximaEPFL/proxima/pull/335/commits/3845991001207e3aba839c1890e902ea4af3ae33)